### PR TITLE
Feature/ability to create&view a page

### PR DESCRIPTION
**Pull request by AlexKapic**
1. without components `toolbar` and `header` created some mock components
2. have some problems with db -- there is no key `name` or `content` for page entity, such keys as a `authorId, workspaceId, parentPageId` have a read-only property
3. "Creates a new content version of the page" is not the stage of current task, so I implemented it as a simple "Create current page"

### CODE REVIEW

File: backend/src/services/page.service.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    const page = new Page();
+    page.authorId = userId;
+    page.workspaceId = workspaceId;
+    page.parentPageId = null;
+    page.childPages = null;
+    page.name = title;
+    page.content = content;
+    const newPage = await connection.manager.save(page);
+
+    return newPage;
```
Comment (aleksandr-kuharenko) [inline backend/src/services/page.service.ts:24]: We should not use models and connections directly in service. Please create a repository for the Page entity and use it in services. Pass object with page content in `pageContents` property

File: backend/src/services/page.service.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+  }).catch(error => {throw new Error(error);});
+};
+
+export const getPages = async (req: Request): Promise<any> => {
```
Comment (aleksandr-kuharenko) [inline backend/src/services/page.service.ts:52]: Please remove all "any" types

File: shared/tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src"],
+  "include": ["."],
```
Comment (aleksandr-kuharenko) [inline shared/tsconfig.json:3]: Why we need this change?

File: frontend/src/store/pages/slice.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const { reducer, actions } = createSlice({
+  name: ReducerName.PAGE,
+  initialState,
+  reducers: {
+    [ActionType.CREATE_PAGE]: (state, action: PayloadAction<IPage>) => {
+      if(state.pages === null) {
+        state.pages = [action.payload];
+      }else{
+        state.pages.push(action.payload);
+      }
```
Comment (aleksandr-kuharenko) [inline frontend/src/store/pages/slice.ts:25]: Run `npm run lint:frontend` and `npm run lint:backend` commands and fix all styles issues

File: backend/package.json [hunk 5]
```diff
@@ -26,13 +26,14 @@
   "devDependencies": {
     "@types/cors": "2.8.12",
     "@types/cryptr": "4.0.1",
-    "nodemon": "2.0.6",
-    "ts-node-dev": "1.0.0",
-    "tscpaths": "0.0.9",
+    "@types/jsonwebtoken": "^8.5.4",
```
Comment (aleksandr-kuharenko) [inline backend/package.json:29]:
```suggestion
    "@types/jsonwebtoken": "8.5.4",
```

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

File: shared/tsconfig.json (hunk 3 above)
Comment (AlexKapic) [reply shared/tsconfig.json:3]: I have some errors in some files during commit -- 
```
0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: =...
The file must be included in at least one of the projects provided
```
and find such solution in google

File: backend/package.json (hunk 5 above)
Comment (AlexKapic) [reply backend/package.json:29]: I have error -- 
```
Could not find a declaration file for module 'jsonwebtoken''
Try `npm install @types/'jsonwebtoken` if it exists or add a new declaration (.d.ts)
```
and find such solution in google

Comment (aleksandr-kuharenko) [reply backend/package.json:29]: Yes, you installed the library with typings but we should lock the version of library. This change will not cause such error again

**Pull request closed without merging**